recipes/sphinxbase/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sphinxbase" %}
+{% set version = "0.8" %}
```

Comment:
I chose the version number based on the [`NEWS`](https://github.com/cmusphinx/sphinxbase/blob/master/NEWS) file in the upstream repo.  I'm happy to change it if someone has a better idea.

Reply:
hmm isn't the actual version `5prealpha`?